src/array_api_stubs/_draft/array_object.py

```diff
@@ -333,12 +341,30 @@ def __dlpack__(
                 not want to think about stream handling at all, potentially at the
                 cost of more synchronizations than necessary.
         max_version: Optional[tuple[int, int]]
-            The maximum DLPack version that the *consumer* (i.e., the caller of
+            the maximum DLPack version that the *consumer* (i.e., the caller of
             ``__dlpack__``) supports, in the form of a 2-tuple ``(major, minor)``.
             This method may return a capsule of version ``max_version`` (recommended
             if it does support that), or of a different version.
             This means the consumer must verify the version even when
             `max_version` is passed.
+        dl_device: Optional[Tuple[Enum, int]]
+            the DLPack device type. Default is ``None``, meaning the exported capsule
+            should be on the same device as ``self`` is. When specified, the format
+            must be a 2-tuple, following that of the return value of :meth:`array.__dlpack_device__`.
+            If the device type cannot be handled by the producer, this function must
+            raise ``BufferError``.
+        copy: Optional[bool]
+            boolean indicating whether or not to copy the input. If ``True``, the
+            function must always copy (paerformed by the producer), potentially allowing
+            data movement across the library (and/or device) boundary. If ``False``,
```

Comment:
this may be slightly misleading, because it's hard to imagine that a copy on the same device is needed to enable crossing a library boundary. Rather, it allows (a) to use in-place operations safely and (b) device transfer.

Reply:
I think copy is copy, it can do whatever this keyword indicates, especially we also are adding a `device` keyword. If 
> copy on the same device is needed to enable crossing a library boundary

is the impression that this text gives, we need to fix it. I'll come up with a longer version (I hate long text if a shorter version does an equally good job) for us to trim/revise.

Reply:
New version in commit b855568ae LGTM, thanks. Resolving.